Gulpfile.js

```diff
@@ -216,3 +216,9 @@ gulp.task('test-client', ['build'], function () {
 });
 
 gulp.task('test', ['lint', 'test-server']);
+
+gulp.task('error-message-lister', function () {
```

Comment:
if I get it right, this tool displays all possible reports' copy and that's it. Designer is something that allows you to create, edit and save things.
So...